src/Stratis.Bitcoin/P2P/PeerConnector.cs

```diff
@@ -270,6 +279,19 @@ public void StartConnectAsync()
             this.logger.LogTrace("(-)");
         }
 
+        private bool PeerIsPartOfExistingGroup(PeerAddress peerAddress)
+        {
+            // Same group if first 16 bits of the network part of IP are equal ie. 192.168.
```

Comment:
Fazz can you add this comment in the method's summary?

Reply:
Yes but it is not very accurate description better to delete it

Reply:
Agreed - its not a good comment.  I've removed it.